evalml/automl/automl_search.py

```diff
@@ -1327,23 +1327,27 @@ def _post_evaluation_callback(
         job_log.write_to_logger(self.logger)
         training_time = evaluation_results["training_time"]
         cv_data = evaluation_results["cv_data"]
+        mean_cv_all_objectives = self._get_mean_cv_scores_for_all_objectives(
+            cv_data,
+            self.objective_name_to_class,
+        )
         cv_scores = evaluation_results["cv_scores"]
         is_baseline = pipeline.model_family == ModelFamily.BASELINE
         mean_cv_score = np.nan if len(cv_scores) == 1 else cv_scores.mean()
         holdout_score = evaluation_results["holdout_score"]
-        if len(cv_scores) == 1 and holdout_score is None:
-            ranking_score = cv_scores[0]
-        elif holdout_score is None:
-            ranking_score = mean_cv_score
+        if holdout_score is None:
+            ranking_additional_objectives = mean_cv_all_objectives
+            if len(cv_scores) == 1:
+                ranking_score = cv_scores[0]
+            else:
+                ranking_score = mean_cv_score
         else:
+            holdout_scores = evaluation_results["holdout_scores"]
+            ranking_additional_objectives = dict(holdout_scores)
```

Comment:
can you clarify for me why we take the holdout score for `ranking_additional_objectives` here?